app/tests/spec/views/confirm.js

```diff
@@ -347,13 +347,13 @@ define(function (require, exports, module) {
                 assert.equal(count, 1);
```

Comment:
nit: Could possibly assert that resend is visible after each submit to ensure that it disappears at correct value?